### ISSUE TITLE
Crash when Windows resumes from suspension

### ISSUE THREAD

**Issue by BrutuZ** [JosefNemec/Playnite#3731]
### Bug Description

Playnite crashes when Windows resumes from suspension mode (S3 Power State).

### To Reproduce

- Launch Playnite 
  - From cold boot: `Playnite.DesktopApp.exe --uridata "playnite://playnite/start/7824d16e-5261-41b5-8a53-b595825717b5" --hidesplashscreen --startclosedtotray`
  - Everything works as intended so far, game is launched, Playnite is ready on the taskbar afterwards.
- Have Windows go into suspension via idle timer
  - Turn off Screen after `1 minute` | Turn off Hard Disk after `20 minutes` | Go to sleep after `2 hours`
- Wake the system
- Try to open Playnite via the tray icon and watch it vanish on hover because the application crashed

### Diagnostics ID

28a9727a-87a1-4ba1-a7c7-cc7dcb32c1b5

**Comment by JosefNemec** [comment]
Can't reproduce. Does it happen when running Playnite in safe mode (main menu -> help -> restart in safe mode)?

**Comment by BrutuZ** [comment]
I don't normally run Playnite in safe mode, it happens when launched with the steps described as I run that daily via Scheduled Tasks.

It doesn't appear to crash when I wake the system right after suspension, but if I stay away for several hours the tray icon will vanish when I hover it and using a desktop shortcut to a game will launch Playnite from scratch, splash screen and all.

**Comment by JosefNemec** [comment]
Add `--safestartup` to those startup arguments and try to reproduce the issue. If it doesn't occur, then it's some addon causing the issue. If it still happens, then no idea. I can't think of anything in Playnite itself that could be causing this.

**Comment by JosefNemec** [comment]
Also since Playnite just terminates without any crash dialog (or crash info in the logs), it means the crash is not happening anywhere in our code (that would result in crash handler dialog). So I literately don't have anything to go by here if I wanted to debug this. It's most likely crash caused by some 3rd party (addon, app, driver or weird system config).

**Comment by BrutuZ** [comment]
It happened again today (just powered on the PC, the Scheduled Task triggered soon after, I closed the game after some 15 minutes and left the system alone until it went to sleep), but not when I tried to deliberately trigger it last time with manual suspension or shorter intervals and regular client launches.

It's a weird and inconsistent crash, since it's killed in its sleep without leaving any log trace I didn't really have high hopes on debugging it. Opened the issue in case the few clues might hint to something but doesn't seem to be the case so I'll just cope.
